src/main/java/com/iexec/core/task/TaskService.java

```diff
@@ -109,7 +110,8 @@ public TaskService(TaskRepository taskRepository) {
     }
 
     public List<Task> getInitializedOrRunningTasks() {
-        return taskRepository.findByCurrentStatus(Arrays.asList(INITIALIZED, RUNNING));
+        return taskRepository.findByCurrentStatus(Arrays.asList(INITIALIZED, RUNNING),
+                Sort.by(Sort.Direction.ASC, "contributionDeadline"));
```

Comment:
How about something like:
`Sort.by(Sort.Direction.ASC, Task.CONTRIBUTION_DEADLINE_FIELD_NAME)`
```
@Field(CONTRIBUTION_DEADLINE_FIELD_NAME)
private Date contributionDeadline;
```
 
 To protect against future updates?

Reply:
Fixed, thanks for the suggestion !